index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+
+    <script defer type="module" src="index.js"></script>    
+
+    <title>test-awesome books</title>
+</head>
+
+<body>
+    <header>
+        <nav class="top-nav">
+            <h1 class="top-h1">Awesome books</h1>
+            <ul class="mylist">
+                <li class="list"><a href="">List</a></li>
+                <li class="add-new"><a href="">Add new</a></li>
+                <li class="contact-us"><a href="">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <div class="our-date"></div>
+    <article class="All-books">
+        <h2 class="books-title">All Awesome book</h2>
+        <section class="container">
+        </section>
+    </article>
+
+    <section class="add-book">
+    <h1>Add a new book</h1>
+        <form>
+            <input class="book-title" type="text" name="tite" id="" placeholder="Title">
+            <input class="author-name" type="text" name="author" id="" placeholder="Author">
```

Comment:
- [ ] [Optional] I would suggest adding HTML validation to your inputs to avoid submitting empty data by the user
![image](https://user-images.githubusercontent.com/89282221/194832750-65ad7a76-0deb-4e98-98f7-028647cb08b4.png)